test/functional/homepage_controller_test.rb

```diff
@@ -58,6 +58,8 @@ class HomepageControllerTest < ActionController::TestCase
     end
 
     should "update latest PopularLinksEdition with changed title and url" do
+      UpdateService.stubs(:call)
```

Comment:
Do we need this stubbing? Commenting it out doesn't seem to cause any ill-effect.

Reply:
this is interesting, this is working without stubbing because, we are stubbing it in test helper [here](https://github.com/alphagov/publisher/blob/cd76211dda73c9bb2dbc96d3c5bb126413cd6f9d/test/test_helper.rb#L52)
not sure, I like this

Reply:
Interesting. I think doing that is OK, so long as we also have test coverage that we're sending the right data (which I think we do).